lib/reviews.js

```diff
@@ -46,11 +59,13 @@ const reviews = (opts) => new Promise((resolve) => {
     opts.page = opts.page || 1;
     opts.country = opts.country || 'us';
 
-    const url = `https://itunes.apple.com/${opts.country}/rss/customerreviews/page=${opts.page}/id=${id}/sortby=${opts.sort}/json`;
+    const url = `https://itunes.apple.com/${opts.country}/rss/customerreviews/page=${opts.page}/id=${id}/sortby=${opts.sort}/xml`;
     return common.request(url, {}, opts.requestOptions);
   })
-  .then(JSON.parse)
-  .then(cleanList);
+  .then((body) => {
+    return xmlParser.parse(body, {attrNodeName: 'attributes', attributeNamePrefix: '', ignoreAttributes: false});
```

Comment:
please wrap this into a function so it keeps the readability of the previous version:
```js
.then(parseXML)
.then(cleanList);
```